scripts/infra.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -euo pipefail
 
+thisdir=`dirname "$0"`
```

Comment:
I actually just ran into this myself, but this fix didn't work for me on Linux....

Can you change it to `thisdir="$(realpath $(dirname "$0"))"`?

Reply:
Done!